contrib/playlib/src/mill/playlib/RouterModule.scala

```diff
@@ -51,9 +65,10 @@ trait RouterModule extends mill.Module with ScalaModule {
   /**
     * The routes compiler type to be used. Can only be one of:
     * <ul>
-    *   <li>[[RouteCompilerType.InjectedGenerator]]
-    *   <li>[[RouteCompilerType.StaticGenerator]]
+    * <li>[[RouteCompilerType.InjectedGenerator]]
```

Comment:
Just a hint: instead of a JavaDoc-style list, in Scaladoc you can simply use `- `.

https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html